src/registrar/admin.py

```diff
@@ -1017,7 +1017,7 @@ class DomainInformationAdmin(ListHeaderAdmin):
                 "description": "Extends type of organization",
                 "fields": [
                     "federal_type",
-                    # "updated_federal_agency",
+                    "updated_federal_agency",
                     # Above field commented out so it won't display
```

Comment:
Are those comments still needed?

Reply:
Yep still needed, will be removed in #1793!